build/index_core_compatible.js

```diff
@@ -1,20 +1,32 @@
+import Assignment from '../es6/assignment';
 import Experiment from '../es6/experiment';
+import * as ExperimentSetup from '../es6/experimentSetup';
 import Interpreter from '../es6/interpreter';
-import RandomPlanoutCoreCompatible from '../es6/ops/randomPlanoutCoreCompatible';
-import Core from '../es6/ops/core';
+import * as Random from '../es6/ops/randomPlanoutCoreCompatible';
+import * as Base from '../es6/ops/base';
+import * as Core from '../es6/ops/core';
+import * as OpsUtils from '../es6/ops/utils';
 import * as Namespace from '../es6/namespace';
-import Assignment from '../es6/assignment';
-import ExperimentSetup from '../es6/experimentSetup';
+
+OpsUtils.registerOperators({
+    "randomFloat": Random.RandomFloat,
+    "randomInteger": Random.RandomInteger,
+    "bernoulliTrial": Random.BernoulliTrial,
+    "bernoulliFilter": Random.BernoulliFilter,
+    "uniformChoice": Random.UniformChoice,
+    "weightedChoice": Random.WeightedChoice,
+    "sample": Random.Sample
+}, true);
 
 export default {
-  Namespace: Namespace,
   Assignment: Assignment,
-  Interpreter: Interpreter,
   Experiment: Experiment,
   ExperimentSetup: ExperimentSetup,
+  Interpreter: Interpreter,
   Ops: {
-    Random: RandomPlanoutCoreCompatible,
-    Core: Core
-  }
+    Random: Random,
+    Core: Core,
+    Base: Base
```

Comment:
Oh, right. I called that out in the commit message (which github conveniently hides by default)

> One trade-off here is that we now expose some previously internal
> modules to clients of the library. In some cases, it seems like the
> right thing to do. For instance, the unit tests demonstrate how to
> register a custom operator, but without PlanOut.Ops.Base one cannot
> replicate that in application code.

An alternative here would be to expose the Base exports from Core as well and no longer expose Base directly.